roles/gluster_volume_create/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# tasks file for gluster_volume_create
+- name: Peer probe the nodes
+  gluster_peer:
+        state: present
+        nodes: "{{ gluster_hosts }}"
+
+- name: Create a volume
+  gluster_volume:
+        state: present
+        volume: "{{ volname }}"
+        bricks: "{{ bricks }}"
+        hosts: "{{ gluster_hosts }}"
+        transport: "{{ transport }}"
+        replica_count: "{{ replica_count }}"
+        arbiter_count: "{{ arbiter_count }}"
+        disperse: "{{ disperse }}"
+        disperse_count: "{{ disperse_count }}"
+        redundancy_count: "{{ redundancy_count }}"
+        force: "{{ force | default('no') }}"
+
+- name: Start the volume if configured
+  gluster_volume:
```

Comment:
Also, volume start I guess is not needed. The gluster_volume takes care of starting the volume.